pallets/gear-eth-bridge/Cargo.toml

```diff
@@ -0,0 +1,104 @@
+[package]
+name = "pallet-gear-eth-bridge"
+description = "TODO (breathx): write pallet's description"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+readme = "README.md"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+# Operational deps.
+log.workspace = true
+parity-scale-codec.workspace = true
+scale-info.workspace = true
+serde = { workspace = true, features = ["derive"], optional = true }
+
+# Gear deps.
+common.workspace = true
+gbuiltin-eth-bridge.workspace = true
+gear-core.workspace = true
+gprimitives.workspace = true
+pallet-gear-builtin.workspace = true
+
+# Substrate deps.
+binary-merkle-tree.workspace = true
+frame-support.workspace = true
+frame-system.workspace = true
+pallet-staking.workspace = true
```

Comment:
The `gear-eth-bridge` pallet doesn't seem to require `pallet-staking` as a dependency but rather as a dev-dependency, does it?